examples/gno.land/p/demo/json/node_test.gno

```diff
@@ -0,0 +1,175 @@
+package json
+
+import (
+	"strconv"
+	"testing"
+)
+
+func TestNodeValue(t *testing.T) {
+    tests := []struct {
+        name  string
+        value interface{}
+		expectType ValueType
+    }{
+        {"string", "test value", String},
+        {"integer", 123, Number},
+        {"float", 1.23, Float},
+        {"boolean", true, Boolean},
+		{"null", nil, Null},
+		{"array", []interface{}{1, 2, 3}, Array},
+		{"object", map[string]interface{}{"key": "value"}, Object},
```

Comment:
weird indentation?

Reply:
consider running `make fmt` in the `/examples` dir

Reply:
well, actually it does not applied any identation in mu IDE windows. So may I misses this weitrd things. anyway, now I apply `fmt` for this file. thanks!